docs/sql-keywords.md

```diff
@@ -19,7 +19,7 @@ license: |
   limitations under the License.
 ---
 
-When `spark.sql.dialect=PostgreSQL` or keep default `spark.sql.dialect=Spark` with setting `spark.sql.dialect.spark.ansi.enabled` to true, Spark SQL will use the ANSI mode parser.
+When `spark.sql.dialect.spark.ansi.enabled` is true, Spark SQL will use the ANSI mode parser.
```

Comment:
I don't think we can do a clean revert, we need to check the diff of this PR and see if they make sense.

Here we should use `spark.sql.ansi.enabled`

Reply:
Yes, my mistake for missing here.